.github/workflows/ci-build-push.yml

```diff
@@ -0,0 +1,34 @@
+name: "build-push"
```

Comment:
Good work Akis. And it is ok for me. What do you think of using make targets in github actions? So at local developer machines, in github actions and gitlab pipelines we will execute same to build and generate artifacts ? 
cc @spirosoik

Reply:
Agreed! 
Rebased and pushed changes !